dist/milligram.css

```diff
@@ -175,6 +174,10 @@ select:focus {
   outline: 0;
 }
 
+select {
+  appearance: menulist;
```

Comment:
This way each browser renders your way. I plan to do another way. It has to have the same behavior in all browsers.